components/omega/doc/devGuide/DataTypesDev.md

```diff
@@ -0,0 +1,52 @@
+(omega-dev-data-types)=
+
+## Data Types and Precision
```

Comment:
Sphinx is complaining about `##` instead of `#` but I think we can leave this as it is for now since this will probably be part of a longer document in the near future.

Reply:
Indeed, I was assuming these would be integrated into another doc so didn't want to presume level-1 headings.